astropy/units/quantity.py

```diff
@@ -1769,21 +1825,25 @@ def _unquantify_allclose_arguments(actual, desired, rtol, atol):
     try:
         desired = desired.to(actual.unit)
     except UnitsError:
-        raise UnitsError("Units for 'desired' ({}) and 'actual' ({}) "
-                         "are not convertible"
-                         .format(desired.unit, actual.unit))
+        raise UnitsError(
+            f"Units for `desired` ({desired.unit}) and `actual` "
```

Comment:
I'd stick with the regular single quotes around 'desired' and 'actual'

Reply:
Actually, this is also why the tests fail - the exception message is explicitly checked.

Reply:
Looks like that was the cause of the test failure!  I suppose I'm getting too used to ReST docstrings...